lib/KDropdownMenu.vue

```diff
@@ -175,19 +201,27 @@
         const popoverIsOpen = popover.clientWidth > 0 && popover.clientHeight > 0;
         // set current element and its siblings
         let focusedElement = document.activeElement;
-        let sibling = focusedElement.nextElementSibling;
-        let prevSibling = focusedElement.previousElementSibling;
+        // let sibling = focusedElement.nextElementSibling;
+        // let prevSibling = focusedElement.previousElementSibling;
+
+        //function to check if the sibling is divider
+        // const isFocusable = element => element && element.tabIndex >= 0;
 
         // manage rotating through the options using arrow keys
         // UP arrow: .keyCode is depricated and should used only as a fallback
         if ((event.key == 'ArrowUp' || event.keyCode == 38) && popoverIsOpen) {
           event.preventDefault();
+
+          // Checking if previous sibling is divider and if yes then skip it
+          let prevSibling = this.getPreviousFocusableSibling(focusedElement);
```

Comment:
lets define these `prevSibling` and `sibling` as const, as we wont need to reassign them anymore.